src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/HandlerOrchestrator.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClientX.Handlers.TransportCreation;
+
+namespace Microsoft.Data.SqlClientX.Handlers
+{
+    internal class HandlerOrchestrator
```

Comment:
I think it would be beneficial:
1) Rename to ConnectionHandlerOrchestrator (since it is specific to connection handling)
2) Move to the recent Handlers/Connection folder

Reply:
Good point, will we have other kinds of orchestrators as well? Will move with the next PR.

Reply:
I'm not entirely sure we *will* have other orchestrators, but I think this one at least needs to indicate it is for connection handling. No problem, moving stuff is likely going to happen at least several times.